odoo/http.py

```diff
@@ -1059,10 +1059,6 @@ def _fix_lang(self, context):
         """
         lang = context.get('lang')
 
-        # inane OpenERP locale
-        if lang == 'ar_AR':
-            lang = 'ar'
```

Comment:
You need to be more explicit than "not used anymore" in your commit message when you make such a change :P

Reply:
Indeed, that was just a quick commit to remove noise from the logs in runbot. It should be clearer now.